Sources/SwiftFormats/FormatStyle+CoreGraphics.swift

```diff
@@ -44,6 +44,32 @@ public struct CGPointParseStrategy: ParseStrategy {
 
 // MARK: -
 
+public extension CGPoint {
+    init(_ string: String) throws {
+        self = try CGPointFormatStyle().parseStrategy.parse(string)
+    }
+
+    init<T, Value>(_ value: Value, format: T) throws where T: ParseableFormatStyle, Value: StringProtocol, T.Strategy == CGPointParseStrategy {
+        self = try format.parseStrategy.parse(value.description)
+    }
+
+    init<T, Value>(_ value: Value, strategy: T) throws where T: ParseStrategy, Value: StringProtocol, T.ParseInput == String, T.ParseOutput == CGPoint {
+        self = try strategy.parse(value.description)
+    }
+}
+
+@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
```

Comment:
I'm avoiding @available declarations right now and relying on a package.swift level declaration of minimum versions. This will need some thought in future but for now it's best to remove...

Reply:
Agreed. Completely a habit born out of living in Playgrounds primarily when I'm playing around with format styles.